src/normalizer/latin.rs

```diff
@@ -31,6 +31,24 @@ impl Normalizer for LatinNormalizer {
         Box::new(Some(token).into_iter())
     }
 
+    fn normalize_with_option<'o>(&self, mut token: Token<'o>, options: NormalizerOption) -> Box<dyn Iterator<Item = Token<'o>> + 'o> {
+        if !token.lemma().is_ascii() {
+            let mut char_map = if options.create_char_map { Some(Vec::new()) } else { None };
+            let mut lemma = String::new();
+            for c in token.lemma().chars() {
+                // if a char can't be deunicoded, skip it.
+                let deunicoded = deunicode_char(c).unwrap_or("").trim();
+                char_map.as_mut().map(|char_map| char_map.push((c.len_utf8() as u8, deunicoded.len() as u8)));
+                lemma.push_str(&deunicoded);
+            }
+            token.lemma = Cow::Owned(lemma);
+            token.char_map = char_map;
+        }
```

Comment:
Here, if  `create_char_map` is set to `false` we can directly use [`deunicode`](https://docs.rs/deunicode/latest/deunicode/) on the whole string instead of `deunicode_char` on each character.